docs/modules/ROOT/pages/documentation/microprofile/config/dynamo-db.adoc

```diff
@@ -0,0 +1,84 @@
+
+IMPORTANT: The type of the database columns for the _key_ and _value_ columns needs to be of type _`String`_ or compatible.
+
+To find the region in the AWS console, check the top right dropdown:
+
+image:microprofile/config/cloud/aws/region.png[AWS region selection]
+
+NOTE: The region name is the snake-case notation, not the full words. E.g. `eu-west-2`.
+
+### From the Admin Console
+
+To configure the config source from the admin console, go to `Configs` -> `your-config` -> `MicroProfile` -> `Config` -> `DynamoDB`.
+
+image:microprofile/config/dynamodb/admin-console-config.png[Payara Server Administration Console configuration route]
+
+From here you configure the name of the database table, the region name, name of the key column containing the key, name of the column containing the value and the maximum number of items to retrieves from the database table.
+
+### From the Command Line
+To configure the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` password aliases from the command line, refer to the xref:documentation/payara-server/password-aliases/password-alias-asadmin-commands.adoc[Password Aliases Asadmin configuration] guide.
+
+To configure the DynamoDB config source from the command line, use the `set-dynamodb-config-source-configuration` asadmin command, specifying the required parameters like this:
+
+[source, shell]
+----
+asadmin> set-dynamodb-config-source-configuration --enabled true --dynamic true --region-name eu-west-2 --table-name Books --key-column-name ISBN --value-column-name Title --limit 10
+----
+You can use the `--enabled` and `--dynamic` options to enable or disable the DynamoDB Config Source on demand.
+
+Also, you can retrieve the current configuration for the DynamoDB Config Source using the `get-dynamodb-config-source-configuration` asadmin command like this:
+
+[source, shell]
+----
+asadmin> get-dynamodb-config-source-configuration
+Enabled  Region Name  Table Name  Key Column Name  Value Column Name  Limit
+true     eu-west-2    Books       ISBN             Title              10
+----
+
+## Usage
+
+Once all of the required options are configured. You should be able to read configuration properties from a database table. For example, let's look at the following diagram. It shows a table named Books with columns ISBN, Author and Title.
+
+image:microprofile/config/dynamodb/database-example.png[Database Example]
+
+If the key column name is set to ISBN and the value column name to Title. You can get the value of the Title column in relation to the ISBN column using the following command along with the ISBN value: 
+
+[source, shell]
+----
+asadmin> get-config-property --source cloud --sourcename dynamodb --propertyName 9780345916419
+The Subtle Knife
+Command get-config-property executed successfully.
+----
+
+NOTE: Currently DynamoDB config source only supports the `get-config-property` command and doesn't support the `set-config-property` and `delete-config-property` commands.
```

Comment:
The "Currently" here is a bit vague. Are there plans to add support for these commands in a future release? If this is the case it might be better to say it so.